types/braintree-web-drop-in/index.d.ts

```diff
@@ -248,6 +250,12 @@ export interface venmoPaymentMethodPayload {
     deviceData?: string | undefined;
 }
 
+export type ActiveView = 'card' | 'paypal' | 'payapCredit' | 'venmo' | 'googlePay' | 'applePay' | 'methods' | 'options' | 'delete-confirmation';
```

Comment:
Have you checked that `payapCredit` is not a typo?
I looked at the documentation and the code, but I wouldn't rule out it being it a typo, since it is only mentioned once:
https://github.com/braintree/braintree-web-drop-in/blob/main/src/dropin.js#L334-L348
(and the documentation is generated based on this)

I couldn't find the code that actually fills the value.

Reply:
I have check it, it is typo. 
The constant should be `paypalCredit`. @joshuajeschek  You can see the implemented here https://github.com/braintree/braintree-web-drop-in/blob/ae2118e45168f5a35e785864e726c7ca7dbd0fa6/src/views/payment-sheet-views/index.js#L11

Reply:
very nice